### ISSUE TITLE
Update Manhattan-compliant model README.rst files

### ISSUE THREAD

**Issue by johnsonb-ucar** [NCAR/DART#111]
## Overview

On the master branch, all of the DART Manhattan-compliant models (plus a few extra models that are only DART Classic-compliant) now have README.rst files in their directories:

```
$ cd ${DARTROOT}
$ git checkout master
$ find . -name README.rst
```

1. ./models/9var/README.rst
2. ./models/am2/README.rst
3. _./models/bgrid_solo/README.rst_
4. ./models/cam-fv/README.rst
5. ./models/cam-old/README.rst
6. ./models/CESM/README.rst
7. ./models/cice/README.rst
8. ./models/cm1/README.rst
9. ./models/coamps_nest/README.rst
10. ./models/ECHAM/README.rst
11. ./models/FESOM/README.rst
12. ./models/forced_lorenz_96/README.rst
13. ./models/gitm/README.rst
14. ./models/LMDZ/README.rst
15. ./models/lorenz_04/README.rst
16. ./models/lorenz_63/README.rst
17. ./models/lorenz_84/README.rst
18. ./models/lorenz_96_2scale/README.rst
19. ./models/lorenz_96/README.rst
20. ./models/mpas_atm/README.rst
21. ./models/noah/README.rst
22. ./models/null_model/README.rst
23. ./models/PBL_1d/README.rst
24. ./models/POP/README.rst
25. ./models/ROMS/README.rst
26. ./models/rose/README.rst
27. ./models/simple_advection/README.rst
28. ./models/wrf_hydro/README.rst
29. ./models/wrf/README.rst

While the READMEs have been converted to reStructuredText, the content contained within the files has not been comprehensively updated.

I have attempted to divide these 29 model directories into three categories: up-to-date, might be up-to-date, certainly out-of-date. It's possible that there is be some nuance of the model mods that I am not familiar with that might cause me to misclassify a given model, so we should collectively look through the documentation to ensure this description is accurate.

### Up-to-date (12 models)

1. 9var
2. am2
3. ECHAM
4. FESOM
5. forced_lorenz_96
6. LMDZ
7. lorenz_04
8. lorenz_63
9. lorenz_84
10. lorenz_96_2_scale
11. lorenz_96
12. rose 

### Might be up-to-date (9 models)

1. bgrid_solo
2. cm1
3. mpas_atm
4. noah
5. null_model
6. PBL_1d
7. simple_advection
8. wrf_hydro
9. wrf

### Certainly out-of-date (8 models)

1. cam-fv
2. cam-old
3. CESM
4. cice
5. coamps_nest
6. gitm
7. POP
8. ROMS

We should look update the 8 README.rst files in this last category as soon as possible. I volunteer to take cice and POP. Like magic, the to-do list instantly got 25% shorter.

If one of the remaining directories is within your purview, to misquote the junior senator from the state of Vermont, "I am once again asking for your documentation support."

**Comment by hkershaw-brown** [comment]
@johnsonb-ucar Good topic for Thursday's software meeting?

* Recommendations for rst editors (I'm using Atom because I like live preview. I'm not sure what the cool kids are using)
* Style guide for documentation, e.g:
    * which `code-block::` for mkmf, fortran, etc. 
    * spelling  e.g. British English
    * use of emojis
    * DART vs. DARTHOME vs. $DARTHOME vs. Dart 

Google has a nice documentation style guide: https://developers.google.com/style

**Comment by johnsonb-ucar** [comment]
Yes, I think this is a good topic for Thursday's software meeting.

Some notes:

- using ``code-block:: perl`` seems to work in Github, but throws an error when using Sphinx, `` WARNING: Could not lex literal_block as "perl". Highlighting skipped.`` Using ``code-block:: [fortran, python, bash]`` works both in Github and Sphinx.
- The emojis don't seem to always make the jump between different hosting platforms (Github, Sphinx, Github pages) but I haven't discerned yet what determines when emojis work and when they don't. 
- Re: DART vs. DARTHOME vs. $DARTHOME vs. Dart vs. DARTROOT vs $DARTROOT. Yes, we're all over the place and should make this uniform.
- Good spelling is important

Reading through the Google documentation style guide now. Thanks!

**Comment by hkershaw-brown** [comment]
There are lots of html files in dart repo.   filter.html has a "getting started/quickstart instructions" which is the 3rd getting started doc in the dart repo. 
* docs/pages/Getting_Started.md
* README.md
* assimilation_code/programs/filter/filter.html

The docs/pages has a bunch of *.md files which were converted from .html. 

I recognize I am adding to the todo list here, but I think the program documentation should be bumped up the priority list:

````
$find assimilation_code/programs -name *.html
````
assimilation_code/programs/obs_seq_verify/obs_seq_verify.html
assimilation_code/programs/wakeup_filter/wakeup_filter.html
assimilation_code/programs/compare_states/compare_states.html
assimilation_code/programs/gen_sampling_err_table/gen_sampling_err_table.html
assimilation_code/programs/perturb_single_instance/perturb_single_instance.html
assimilation_code/programs/system_simulation/system_simulation.html
assimilation_code/programs/compute_error/compute_error.html
assimilation_code/programs/preprocess/preprocess.html
assimilation_code/programs/obs_impact_tool/obs_impact_tool.html
assimilation_code/programs/create_fixed_network_seq/create_fixed_network_seq.html
assimilation_code/programs/obs_loop/obs_loop.html
assimilation_code/programs/perfect_model_obs/perfect_model_obs.html
assimilation_code/programs/obs_selection/obs_selection.html
assimilation_code/programs/obs_sequence_tool/obs_sequence_tool.html
assimilation_code/programs/integrate_model/integrate_model.html
assimilation_code/programs/obs_diag/oned/obs_diag.html
assimilation_code/programs/obs_diag/threed_cartesian/obs_diag.html
assimilation_code/programs/obs_diag/threed_sphere/obs_diag.html
assimilation_code/programs/fill_inflation_restart/fill_inflation_restart.html
assimilation_code/programs/obs_seq_coverage/obs_seq_coverage.html
assimilation_code/programs/advance_time/advance_time.html
assimilation_code/programs/model_mod_check/model_mod_check.html
assimilation_code/programs/closest_member_tool/closest_member_tool.html
assimilation_code/programs/restart_file_tool/restart_file_tool.html
assimilation_code/programs/filter/filter.html
assimilation_code/programs/obs_keep_a_few/obs_keep_a_few.html
assimilation_code/programs/create_obs_sequence/create_obs_sequence.html
assimilation_code/programs/obs_seq_to_netcdf/obs_seq_to_netcdf.html
assimilation_code/programs/obs_common_subset/obs_common_subset.html

**Comment by kdraeder** [comment]
I'll work on Certainly out-of-date: 1. cam-fv and maybe 3. CESM (see below).

Is there a summary of the Google style that we'll use,
or should I plow through the whole guide on my own?
(and write a summary?)

Questions of context: 
Ben, have you worked on the POP andor cice READMEs yet?
AKA should I start with those as a style template (the committed ones look old)?
Should there be a CESM+DART overview in the models/**cesm**/README.rst,
for reference by the component READMEs?
Do we want to make all of the CESM READMEs have a consistent look, 
beyond what we settle on for style for all READMEs?

I don't see clm in the lists of directories.

**Comment by hkershaw-brown** [comment]
@kdraeder For the first question, let's go with getting the content of the documentation up to date as the first milestone, then use the style guide as a tool in the review stage.  Having said that, the [writing for a global audience](https://developers.google.com/style/translation) is a short read with some helpful tips.
